### ISSUE TITLE
⛔ Scroll Snap doesn't work properly in Safari with fixed background images

### ISSUE THREAD

**Issue by maxjaenicke** [elementor/elementor#29788]
### Prerequisites

- [X] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [X] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

When a container has a fixed background image the scroll snap feature behaves not as intended in Safari. Scrolling is jittery.
https://staging.immakerlab.de/testscrollsnap/

### Steps to reproduce

1. Create at least two full-height containers. Add a background image and choose "Attachment: fixed"
2. Enable Scroll Snap
3. Open in Safari Version 18.2 (20620.1.16.11.8)

### Expected behavior

1. Containers should snap to designated position

### Isolating the problem

- [X] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [X] This bug happens with the Hello Elementor theme active.
- [X] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

<details>
<summary>Click to reveal</summary>

````txt


== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: Debian 11 v10.5.26
	PHP Version: 8.2.26
	PHP Memory Limit: 256M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 50M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.4.5
	Site URL: 
	Home URL: 
	WP Multisite: No
	Max Upload Size: 50 MB
	Memory limit: 40M
	Max Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: en_US
	Timezone: Europe/Berlin
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.4.2
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en-US
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36 Edg/131.0.0.0

== Active Plugins ==
	Elementor
		Version: 3.26.3
		Author: Elementor.com

	Elementor Pro
		Version: 3.26.2
		Author: Elementor.com


== Elements Usage ==
	
	archive : 1
		heading : 8
		image : 1
		posts : 1
	footer : 4
		animated-headline : 2
		button : 2
		form : 1
		heading : 15
		icon-list : 1
		image : 27
		text-editor : 14
	header : 1
		nav-menu : 1
		search-form : 1
		theme-site-logo : 1
	page : 15
		animated-headline : 2
		button : 4
		counter : 3
		gallery : 3
		heading : 32
		image : 32
		lottie : 2
		nav-menu : 2
		post-info : 1
		posts : 2
		spacer : 2
		table-of-contents : 1
		text-editor : 20
		theme-post-content : 1
		theme-post-title : 2
	popup : 12
		animated-headline : 1
		button : 1
		divider : 1
		gallery : 5
		heading : 19
		icon : 6
		image : 8
		spacer : 1
		text-editor : 12
		video : 4
	section : 35
		animated-headline : 2
		button : 1
		counter : 9
		divider : 3
		flip-box : 10
		gallery : 2
		heading : 125
		hotspot : 1
		image : 80
		image-box : 6
		image-carousel : 4
		lottie : 1
		post-info : 1
		posts : 1
		shortcode : 1
		spacer : 8
		text-editor : 64
		theme-post-featured-image : 1
		theme-post-title : 1
		video : 7
	wp-post : 12
		counter : 44
		gallery : 36
		heading : 210
		image : 16
		text-editor : 30
		video : 2
	wp-page : 29
		animated-headline : 1
		button : 42
		counter : 44
		divider : 2
		form : 1
		gallery : 22
		heading : 440
		hotspot : 3
		html : 9
		icon : 2
		icon-list : 1
		image : 151
		image-carousel : 20
		lottie : 21
		nav-menu : 2
		posts : 2
		shortcode : 1
		spacer : 4
		text-editor : 240
		video : 31
	single-post : 3
		button : 2
		heading : 2
		image : 2
		post-info : 1
		posts : 1
		spacer : 2
		table-of-contents : 1
		text-editor : 1
		theme-post-content : 1
		theme-post-title : 1
	search-results : 1
		archive-posts : 1
		nav-menu : 1
		search-form : 1
		theme-archive-title : 1
		theme-site-logo : 1
	error-404 : 1
		heading : 2


== Settings ==
	
	cpt_support: post, page
	allow_tracking: yes
	unfiltered_files_upload: 1


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	
	google_maps: Active


== Elementor Experiments ==
	Inline Font Icons: Inactive by default
	Additional Custom Breakpoints: Active
	Container: Inactive by default
	Upgrade Swiper Library: Active by default
	Nested Elements Performance: Active by default
	Optimized Markup: Inactive by default
	Plugin Onboarding: Active by default
	CSS Smooth Scroll: Active by default
	Head Loading Styles: Inactive by default
	Default to New Theme Builder: Active by default
	Hello Theme Header & Footer: Inactive by default
	Elementor Home Screen: Active by default
	Landing Pages: Active by default
	Nested Elements: Active by default
	Editor Top Bar: Active by default
	Pages Panel: Inactive by default
	Element Caching: Inactive by default
	Link In Bio: Active by default
	Floating Buttons: Active by default
	Elementor Editor Events: Inactive by default
	Atomic Widgets: Inactive by default
	Global Classes: Inactive by default
	Launchpad Checklist: Inactive by default
	Menu: Inactive by default


== Log ==
	
PHP: showing 11 of 11PHP: 2022-10-07 15:45:32 [error X 1][../elementor-pro/modules/theme-builder/classes/conditions-manager.php::286] Uncaught Error: Call to a member function update_meta() on null in ../elementor-pro/modules/theme-builder/classes/conditions-manager.php:286
Stack trace:
#0 ../elementor-pro/modules/theme-builder/classes/conditions-manager.php(167): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager-&gt;save_conditions(1536, Array)
#1 [internal function]: ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager-&gt;ajax_save_theme_template_conditions(Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#2 ../elementor/core/common/modules/ajax/module.php(171): call_user_func(Array, Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#3 ../wp-includes/class-wp-hook.php(307): Elementor\Core\Common\Modules\Ajax\Module-&gt;handle_ajax_request(&#039;&#039;)
#4 ../wp-includes/class-wp-hook.php(331): WP_Hook-&gt;apply_filt [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-10-11 14:13:47 [warning X 1][../elementor/core/files/file-types/svg.php::234] simplexml_load_file(): I/O warning : failed to load external entity &quot;../wp/wp-content/uploads/2022/10/Triangle.svg&quot; [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-10-12 14:57:31 [notice X 6][../elementor/includes/conditions.php::87] Undefined index: motion_fx_transform_x_anchor_point [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-10-27 17:42:05 [error X 1][../elementor-pro/modules/popup/module.php::102] Uncaught Error: Call to undefined method Elementor\Core\Kits\Documents\Kit::save_display_settings_data() in ../elementor-pro/modules/popup/module.php:102
Stack trace:
#0 [internal function]: ElementorPro\Modules\Popup\Module-&gt;save_display_settings(Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#1 ../elementor/core/common/modules/ajax/module.php(171): call_user_func(Array, Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#2 ../wp-includes/class-wp-hook.php(307): Elementor\Core\Common\Modules\Ajax\Module-&gt;handle_ajax_request(&#039;&#039;)
#3 ../wp-includes/class-wp-hook.php(331): WP_Hook-&gt;apply_filters(&#039;&#039;, Array)
#4 ../wp-includes/plugin.php(476): WP_Hook-&gt;do_action(Array)
#5 ../wp-admin/admin-ajax.php(187): do_action(&#039;wp_ajax_element...&#039;)
#6 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-10-28 17:11:53 [notice X 1][../elementor/includes/template-library/sources/local.php::747] ob_end_clean(): failed to delete buffer. No buffer to delete [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-09-05 18:01:18 [notice X 1][../elementor-pro/license/updater.php::84] Undefined index: new_version [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-11-14 17:00:49 [notice X 21778][../elementor-pro/modules/forms/submissions/database/query.php::103] Using ${var} in strings is deprecated, use {$var} instead [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-11-24 09:58:01 [warning X 6][../elementor-pro/modules/dynamic-tags/acf/tags/acf-color.php::34] Undefined array key 1 [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-11-24 10:23:52 [notice X 5][../elementor/core/experiments/manager.php::163] version_compare(): Passing null to parameter #2 ($version2) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-11-24 10:25:18 [warning X 7][../elementor-pro/modules/posts/skins/skin-content-base.php::265] Attempt to read property &quot;ID&quot; on null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-11-27 09:44:07 [error X 40][../elementor/core/wp-api.php::26] Uncaught Error: Call to undefined function Elementor\Core\get_plugins() in ../elementor/core/wp-api.php:26
Stack trace:
#0 ../elementor/modules/compatibility-tag/base-module.php(96): Elementor\Core\Wp_Api-&gt;get_plugins()
#1 ../elementor/modules/compatibility-tag/base-module.php(58): Elementor\Modules\CompatibilityTag\Base_Module-&gt;get_plugins_with_header()
#2 ../elementor/modules/compatibility-tag/module.php(57): Elementor\Modules\CompatibilityTag\Base_Module-&gt;get_plugins_to_check()
#3 ../elementor/modules/compatibility-tag/base-module.php(148): Elementor\Modules\CompatibilityTag\Module-&gt;get_plugins_to_check()
#4 ../wp-includes/class-wp-hook.php(324): Elementor\Modules\CompatibilityTag\Base_Module-&gt;Elementor\Modules\CompatibilityTag\{closure}()
#5 ../wp-includes/class-wp-hook.php(348): WP_Hook-&gt;apply_filters()
#6 ../wp-includes/plugin.php(517): WP_Hook-&gt;do_action()
#7 ../elementor/modules/system-info/module.php(340): do_action()
#8 ../elementor/includes/tracker.php(241): Elementor\Modules\System_Info\Module::get_allowed_reports()
#9 ../elementor/includes/tracker.php(530): Elementor\Tracker::get_system_reports_data()
#10 ../elementor/includes/tracker.php(141): Elementor\Tracker::get_tracking_data()
#11 ../wp-includes/class-wp-hook.php(324): Elementor\Tracker::send_tracking_data()
#12 ../wp-includes/class-wp-hook.php(348): WP_Hook-&gt;apply_filters()
#13 ../wp-includes/plugin.php(565): WP_Hook-&gt;do_action()
#14 ../wp-cron.php(191): do_action_ref_array()
#15 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 2 of 2JS: 2022-10-13 08:35:41 [error X 3][../elementor/assets/js/editor.min.js?ver=3.7.8:3:646957] this.model.isValidChild is not a function. (In \&#039;this.model.isValidChild(C)\&#039;, \&#039;this.model.isValidChild\&#039; is undefined) 
JS: 2022-10-27 09:24:55 [error X 1][../elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: \&quot;view3182\&quot;) has already been destroyed and cannot be used. 

Log: showing 20 of 302023-11-24 09:52:21 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.7.8',
  'to' => '3.17.3',
)]
2023-11-24 10:24:20 [info] elementor-pro::elementor_pro_updater Started 
2023-11-24 10:24:20 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-11-24 10:24:20 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-11-24 10:24:20 [info] Elementor Pro/Upgrades - _v_3_7_2_woocommerce_rename_related_to_related_products Start  
2023-11-24 10:24:21 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.6.2',
  'to' => '3.17.1',
)]
2023-11-24 10:24:21 [info] Elementor Pro/Upgrades - _v_3_7_2_woocommerce_rename_related_to_related_products Finished 
2023-11-24 10:24:21 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.6.2',
  'to' => '3.17.1',
)]
2025-01-06 11:31:13 [info] elementor::elementor_updater Started 
2025-01-06 11:31:13 [info] Elementor/Upgrades - _on_each_version Start  
2025-01-06 11:31:14 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.17.3',
  'to' => '3.26.3',
)]
2025-01-06 11:31:14 [info] Elementor/Upgrades - _on_each_version Finished 
2025-01-06 11:31:14 [info] Elementor/Upgrades - _v_3_26_0 Start  
2025-01-06 11:31:14 [info] Elementor/Upgrades - _v_3_26_0 Finished 
2025-01-06 11:31:14 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.17.3',
  'to' => '3.26.3',
)]
2025-01-06 11:33:52 [info] elementor-pro::elementor_pro_updater Started 
2025-01-06 11:33:52 [info] Elementor Pro/Upgrades - _on_each_version Start  
2025-01-06 11:33:52 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2025-01-06 11:33:52 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.17.1',
  'to' => '3.26.2',
)]
2025-01-06 11:33:52 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.17.1',
  'to' => '3.26.2',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==

````

</details>

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by nicholaszein** [comment]
Hi @maxjaenicke,

Thank you for reaching out! 🙏 

### ⛔ While reviewing your report we concluded this is not an issue caused by Elementor. Here's what we found out:
We tested with different sites, different machines, and different settings, and in all situations the page didn't behave _"jittery"_.

Your issue is most likely caused by a GPU bottleneck, or rendering problems with Safari, which usually has problems with animations and layers. Since fixed positioned background images do require a level of animation and generate a new layer context, it might be exactly what is causing your issue. However, if it was an inherited problem, we would have been able to reproduce it on our end.

Keep in mind that the Scroll Snap implementation used by Elementor is 100% CSS-based. Therefore, there isn't much we can do about browser behavior. It is usually recommended to avoid using Scroll Snap with animations and scroll-based motion effects (JS).

It is also important to note that if you're not experiencing any animation with the Scroll Snap feature enabled, it might mean your OS (Operating System) or your browser have Reduced Motion enabled. In this case any animation, including Smooth Scroll won't be triggered. Animations and Smooth Scroll respect the user's preferences for Reduced Motion set at the OS level.

### ✅ What you can do to solve your issue:
Try to avoid applying complex effects such as the ones described above.

### 👍 For now, that's all we can help with. 

If you think this is still an issue caused by Elementor after following the troubleshooting steps above, feel free to open a new 🐞 [Bug Report](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml).

Kind regards
